extension/src/test/e2e/wdio.conf.ts

```diff
@@ -1,13 +1,30 @@
 import { join, resolve } from 'path'
+import { mkdirp } from 'fs-extra'
 import { Options } from '@wdio/types'
+import { getVenvBinPath } from '../../python/path'
+import { Logger } from '../../common/logger'
+
+const screenshotDir = join(__dirname, 'screenshots')
 
 export const config: Options.Testrunner = {
   after: async function () {
     await browser.switchToFrame(null)
     await browser.switchToFrame(null)
   },
+  afterTest: async (test, __, { passed }) => {
```

Comment:
[F] This is close to what is provided in the example tests for the service. If I get time I will extend the way that we do timeouts to also take screenshots.